src/app/shared/services/tag-manager.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core';
+
+
+export enum EventCategory {
+  Sell = 'Sell',
```

Comment:
I think that enum entries, since they are constants, should be upper case with underscore.

Reply:
Uhm, https://www.typescriptlang.org/docs/handbook/enums.html

Keys should be camelcased, values are constrained by GA events requirements